test/common/grpc/grpc_client_integration.h

```diff
@@ -150,6 +173,11 @@ class DeltaSotwIntegrationParamTest
   testing::Combine(testing::ValuesIn(TestEnvironment::getIpVersionsForTest()),                     \
                    testing::ValuesIn(TestEnvironment::getsGrpcVersionsForTest()),                  \
                    testing::Values(Grpc::LegacyOrUnified::Legacy, Grpc::LegacyOrUnified::Unified))
-
+#define EDS_MODE_DELTA_SOTW_GRPC_CLIENT_INTEGRATION_PARAMS                                         \
```

Comment:
integration test for eds over grpc has cardinality of 16, it takes some time to run it.

Reply:
Can these tests be sharded?

Reply:
yes

Reply:
after extracting eds grpc test to separate file, test running time have decreased, don't think we need to shard